lib/src/models/app_state.dart

```diff
@@ -6,18 +6,17 @@ class AppState {
 
   User _user;
   User get user => _user;
-  
-  Function(User) get setUser =>  _userController.sink.add;
 
-  AppState()
-  {
-    _userController.listen((onData){
+  Function(User) get setUser => _userController.add;
```

Comment:
I understand that this was changed because there was an error with the login. But this method setUser was setting the user in the sink of the _userController, which will notify any subscribers to that stream. My memory could fail me, but that was the idea

Reply:
YEah there's is issue in sink getter. So was changed